hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -75,6 +76,12 @@
     private static final long ON_MEMBER_LEFT_DELAY_MILLIS = 1111;
     private static final int HEARTBEAT_CALL_TIMEOUT_RATIO = 4;
     private static final long MAX_DELAY_MILLIS = SECONDS.toMillis(10);
+    // the maximum deviation between operation-heartbeat packets compared to the heartbeat frequency.
+    // if e.g. the heartbeat is expected every 15 seconds, and after 25 seconds a heartbeat is received,
+    // then the deviation is 100*((25-15)/15)=67%
+    // If set to 0, there will not be any detection of qos violations.
+    private static final int MAX_DEVIATION_PERCENTAGE = Integer.getInteger("hazelcast.diagnostics.tcp.qos.deviation", 33);
```

Comment:
Can you make this a `new HazelcastProperty(PREFIX + ".tcp.qos.deviation", 33)`, so it's easier to set the property (e.g. via `config.setProperty()`).